content/en/infrastructure/serverless/datadog_lambda_layer.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Datadog Lambda Layer
+kind: documentation
+aliases:
+  - infrastructure/serverless/datadog-lambda-layer
+  - infrastructure/serverless/lambda_layer
+  - infrastructure/serverless/lambda-layer
+further_reading:
+- link: "/integrations/amazon_lambda/"
+  tag: "AWS Lambda Integration"
+  text: "AWS Lambda Integration"
+---
+
+The Datadog Lambda Layer is responsible for:
+
+- Generating real-time [enhanced Lambda metrics][1] for invocations, errors, cold starts, etc.
+- Submitting custom metrics (synchronously and asynchronously)
+- Automatically propagating tracing headers from upstream requests to downstream services. This enables full distributed tracing across Lambda functions, hosts, containers, and other infrastructure running the Datadog Agent.
+
+Datadog offers Lambda Layers for Python, Node.js, and Ruby. Go is also supported with a [package][7] to include in your project. Datadog is working on support for new languages and runtimes; if there is another runtime you would like Datadog to support, reach out to the [Datadog support team][8].
+
+# Setup
+
+## AWS Console
+
+The Datadog Lambda Layer ARN includes a region, language runtime, and version. Construct yours in the following format:
+
+```text
+arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
+```
+
+For example:
+
+```text
+arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Python37:11
+```
+
+| Language | Runtime                                        | Releases             |
+| -------- | ---------------------------------------------- | -------------------- |
+| Python   | `Python27`, `Python36`, `Python37`, `Python38` | [Latest release][3] |
+| Node.js  | `Node8-10`, `Node10-x`, `Node12-x`             | [Latest release][4] |
+| Ruby     | `Ruby`                                         | [Latest release][5] |
+
+**Golang:** Since Go binaries are statically-linked, Datadog offers a [package][7] you can import into your project. No Lambda layer is required.
```

Comment:
```suggestion
**Golang:** Since Go binaries are statically linked, Datadog offers a [package][7] you can import into your project. No Lambda Layer is required.
```